### DISCUSSION TITLE
Onedrive su elementary os 8

### DISCUSSION THREAD

**Discussion by Fede1317** [elementary/triage#638] [Q&A]
Ciao a tutti, ho urgente bisogno di configurare onedrive su elementary 8 ma sto riscontrando enormi difficoltà.
Chi potrebbe aiutarmi a riguardo?

**Comment by teamcons** [comment]
Ciao

Dai qualche informazione sul tuo problema, possibilmente in inglese e con il maggior numero di dettagli possibile

e forse qualcuno troverà una soluzione

**Reply by Fede1317** [reply]
good morning, could someone please tell me the procedure to configure onedrive from terminal on elementary os 8? I found the procedure for ubuntu and others online, but not for elementary.
I need access to the folders I use at work.
Thanks.

**Reply by teamcons** [reply]
Good morning

> good morning, could someone please tell me the procedure to configure onedrive from terminal on elementary os 8? I found the procedure for ubuntu and others online, but not for elementary. I need access to the folders I use at work. Thanks.

elementary is based on Ubuntu. It is mostly an interface on top of it

Any guide for the terminal that works for Ubuntu, works for elementary
